testing/typescript/data/stores.ts

```diff
@@ -1,12 +1,12 @@
 /* eslint-disable no-new */
-/* eslint-disable @typescript-eslint/no-unused-vars */
-/* eslint-disable @typescript-eslint/no-unused-vars-experimental */
-/* eslint-disable @typescript-eslint/explicit-function-return-type */
-/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
 // eslint-disable-next-line import/no-extraneous-dependencies
 import $ from 'jquery';
 import { HttpClient, HttpHandler } from '@angular/common/http';
 import CustomStore from '../../../js/data/custom_store';
+import Store from '../../../js/data/abstract_store';
```

Comment:
```suggestion
import { Store } from '../../../js/data';
```

This required updating to the 22_1 branch, I've just merged #20807